api/src/test/java/uk/gov/dhsc/htbhf/claimant/PaymentCycleIntegrationTests.java

```diff
@@ -303,14 +350,14 @@ private void assertPaymentCycleIsFullyPaid(PaymentCycle paymentCycle, List<Local
         assertThat(payment.getPaymentAmountInPence()).isEqualTo(expectedVoucherEntitlement.getTotalVoucherValueInPence());
     }
 
-    private void assertPaymentCycleHasNoPayment(PaymentCycle paymentCycle, PaymentCycleVoucherEntitlement expectedVoucherEntitlement) {
+    private void assertPaymentCycleHasNoPayment(PaymentCycle paymentCycle, List<LocalDate> childrenDob) {
```

Comment:
Could possibly be renamed to `assertPaymentCycleWithNoPayment` as it does a bit more than just checking that no payment exists.